README.md

```diff
@@ -25,7 +25,56 @@ with API token or Jenkins CLI access but can also appear with features
 such as the [Authorize Project
 plugin](http://plugins.jenkins.io/authorize-project/)
 or other plugins that require details of user permissions or group
-membership outside of a user's web session)
+membership outside of a user's web session).
+
+### Compatibility Notes
+
+Various LDAP servers use different operational attributes to make decisions on
+and expose configurations of concepts such as disabling an account, locking an
+account, and specifying a time interval the account is valid for. These policies
+are normally enforced by the LDAP server itself when performing user authentication.
+Once a user has configured API tokens or SSH keys, these parallel authentication
+mechanisms can only make limited policy decisions based on checking for known user
+attributes without requiring the user to re-enter their password. These systems include
+[**slapo-ppolicy**(5)](https://linux.die.net/man/5/slapo-ppolicy) from OpenLDAP,
+Active Directory Application Mode (ADAM), classic Active Directory, and eDirectory LDAP.
+The support of these features is entirely dependent upon the LDAP server implementation
+properly exposing these operational attributes which is dependent on the exact version
+and distribution of the LDAP server in use.
```

Comment:
I suggest expanding this a bit to say more about how these capabilities may not work depending on whether the LDAP server implements any of them, how it implements them, how they are configured, and how they are exposed.